.rubocop.yml

```diff
@@ -23,7 +24,7 @@ Metrics/ClassLength:
   Description: Avoid classes longer than 100 lines of code.
   Enabled: true
   CountComments: false
-  Max: 100
+  Max: 200
```

Comment:
I wouldn't change this, just exclude this work temporarily from the rule. Also the Description is now out of date